modules/tinymce/src/plugins/help/main/ts/ui/KeyboardNavTab.ts

```diff
@@ -3,68 +3,90 @@ import { Dialog } from 'tinymce/core/api/ui/Ui';
+
+<p>To move from one UI section to the previous, press <strong>Shift+Tab</strong>.</p>
+
+<p>The <strong>Tab</strong> order of these UI sections is:
+
+<ol>
+<li>Menu bar</li>
+<li>Each toolbar group</li>
+<li>Sidebar</li>
+<li>Element path in the footer</li>
+<li>Word count toggle button in the footer</li>
+<li>Branding link in the footer</li>
+<li>Editor resize handle in the footer</li>
+</ol>
 
-<p>Pressing shift + tab will move backwards through the same sections, except when moving from the footer to the toolbar. Focusing the element path then pressing shift + tab will move focus to the first toolbar group, not the last.</p>
+<p>If a UI section is not present, it is skipped.</p>
 
-<h2>Moving within UI sections</h2>
+<p>If the footer has keyboard navigation focus, and there is no visible sidebar, pressing <strong>Shift+Tab</strong> moves focus to the first toolbar group, not the last.
+
+<h1>Navigate within UI sections</h1>
+
+<p>To move from one UI element to the next, press the appropriate <strong>Arrow</strong> key.</p>
+
+<p>The <strong>Left</strong> and <strong>Right</strong> arrow keys</p>
 
-<p>Keyboard navigation within UI sections can usually be achieved using the left and right arrow keys. This includes:</p>
 <ul>
-  <li>moving between menus in the menubar</li>
-  <li>moving between buttons in a toolbar group</li>
-  <li>moving between items in the element path</li>
+<li>move between menus in the menu bar.</li>
+<li>open a sub-menu in a menu.</li>
+<li>move between buttons in a toolbar group.</li>
+<li>move between items in the footer’s element path.</li>
 </ul>
 
-<p>In all these UI sections, keyboard navigation will cycle within the section. For example, focusing the last button in a toolbar group then pressing right arrow will move focus to the first item in the same toolbar group. </p>
+<p>The <strong>Down</strong> and <strong>Up</strong> arrow keys
+
+<ul>
+<li>move between menu items in a menu.</li>
+<li>move between items in a toolbar pop-up menu.</li>
+</ul>
 
-<h1>Executing buttons</h1>
+<p><strong>Arrow</strong> keys cycle within the focused UI section.</p>
 
-<p>To execute a button, navigate the selection to the desired button and hit space or enter.</p>
+<p>To close an open menu, an open sub-menu, or an open pop-up menu, press the <strong>Esc</strong> key.
 
-<h1>Opening, navigating and closing menus</h1>
+<p>If the current focus is at the ‘top’ of a particular UI section, pressing the <strong>Esc</strong> key also exits keyboard navigation entirely.</p>
 
-<p>When focusing a menubar button or a toolbar button with a menu, pressing space, enter or down arrow will open the menu. When the menu opens the first item will be selected. To move up or down the menu, press the up or down arrow key respectively. This is the same for submenus, which can also be opened and closed using the left and right arrow keys.</p>
+<h1>Execute a menu command or toolbar button</h1>
```

Comment:
Just wondering if 'menu item` or `menu button` might be clearer than `menu command`?

Reply:
Works for me: done.